src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/GraphCreation/PXGraphCreationInGraphSemanticModelTests.cs

```diff
@@ -12,7 +13,7 @@ namespace Acuminator.Tests.Tests.StaticAnalysis.GraphCreation
     public class PXGraphCreationInGraphSemanticModelTests : DiagnosticVerifier
     {
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() =>
-            new PXGraphAnalyzer(new PXGraphCreationInGraphSemanticModelAnalyzer());
+            new PXGraphAnalyzer(CodeAnalysisSettings.Default, new PXGraphCreationInGraphSemanticModelAnalyzer());
```

Comment:
Don't you need `WithRecursiveAnalysisEnabled()` here? Tests should not rely on the default settings because they can change in the future; all settings that are used in the tests must be set explicitly.

Reply:
Ok